lib/api/protocol.js

```diff
@@ -440,6 +440,33 @@ module.exports = function(Nightwatch) {
   };
 
   /**
+   * Click at the current mouse coordinates (set by moveto).
+   *
+   * The button can be (0, 1, 2) or ('left', 'middle', 'right'). It defaults to left mouse button, and if you don't pass in a button but do pass in a callback, it will handle it correctly.
+   *
+   * @link /session/:sessionId/click
+   * @param {string|number} button The mouse button
+   * @param {function} [callback] Optional callback function to be called when the command finishes.
+   * @api protocol
+   */
+  Actions.mouseButtonClick = function(button, callback) {
+    var buttonIndex;
+    if (typeof button === 'string') {
+      buttonIndex = [
+        MOUSE_BUTTON_LEFT,
+        MOUSE_BUTTON_MIDDLE,
+        MOUSE_BUTTON_RIGHT
+      ].indexOf(button.toLowerCase());
+
+      if (buttonIndex !== -1) {
+        button = buttonIndex;
+      }
+    }
+
+    return postRequest('/click', {button: button}, callback);
```

Comment:
It looks like the button is always required. You should be able to do just `.mouseButtonClick()`.